pravega/zookeeper-operator
# Issue: 42: Clean up PVC after cluster is deleted

### Changelog description
This PR adds a finalizer to delete PVCs after the zk cluster is deleted. Users can decide whether to enable this finalizer by configuring it in the yaml file. The operator DOES NOT delete the PVC by default, users have to explicitly enable it.

To enable:
```
...
persistence:
  reclaimPolicy: Delete
...
```

### Purpose of the change
Fix #42 

### How to test the change
- Use this branch and start the operator
- Create a zk cluster
- Delete the zk cluster
- Verify that the PVC is deleted

Signed-off-by: wenqi <Wenqi.Mou@emc.com>
